test/functional/test_framework/blocktools.py

```diff
@@ -51,19 +55,29 @@
 # From BIP141
 WITNESS_COMMITMENT_HEADER = b"\xaa\x21\xa9\xed"
 
+NORMAL_GBT_REQUEST_PARAMS = {"rules": ["segwit"]}
 
-def create_block(hashprev, coinbase, ntime=None, *, version=1):
+
+def create_block(hashprev=None, coinbase=None, ntime=None, *, version=None, tmpl={}, txlist=None):
```

Comment:
Passing in a dictionary as a default parameter value is triggering the linter.

Reply:
So how do we fix the linter?

Reply:
Something like this?

```suggestion
def create_block(hashprev=None, coinbase=None, ntime=None, *, version=None, tmpl=None, txlist=None):
  tmpl = {} if tmpl is None else tmpl
```

Reply:
or use the suggestion from the linter

Reply:
There isn't a problem with this PR. The problem is the linter being wrong.

Reply:
Changing this to `None` here since it conceptually makes more sense anyway, but in other cases the linter should probably be fixed instead.